corva/testing.py

```diff
@@ -0,0 +1,78 @@
+import contextlib
+import functools
+import os
+from unittest import mock
+
+import fakeredis
+import pytest
+
+from corva.configuration import Settings
+
+
+@pytest.fixture(scope='function')
```

Comment:
maybe it'd be better to set `autouse=True`, so that users even dont know about the fixture